### PR TITLE
Make `properties` an optional parameter

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Closure Compiler friendly!

### CODE REVIEW

File: paper-toast.html [hunk 1]
```diff
@@ -192,7 +192,7 @@
 
         /**
          * Show the toast. Without arguments, this is the same as `open()` from `IronOverlayBehavior`.
-         * @param {Object|string|undefined} properties Properties to be set before opening the toast.
+         * @param {Object|string} [properties] Properties to be set before opening the toast.
```
Comment (rictic) [inline paper-toast.html]: I think that the syntax for making a param optional is `{Object=|string=}` or maybe `{(Object|string)=}`

See: https://developers.google.com/closure/compiler/docs/js-for-compiler#types

Comment (valdrinkoshi) [inline paper-toast.html]: true, updated!

**Comment by valdrinkoshi** [comment]
@rictic updated, ptal :eyes:

**Comment by rictic** [comment]
LGTM!

**Pull request merged**